SimpleSound/script.json

```
@@ -0,0 +1,21 @@
+{
+    "name": "Simple Sound",
+    "script": "simplesound.js",
+    "version": "2.1",
+    "description": "The goal of this script is to play/stop sound effects from the Roll20 Jukebox via macro and chat commands.",
```

```suggestion
    "description": "The goal of this script is to play/stop sound effects from the Roll20 Jukebox via macro and chat commands.\n\nCommands: `!splay trackname`, `!sstop trackname`, `!sstop`(stops all), `!swhisper`(toggle the GM whisper status)\n\nSee [Readme](https://github.com/Roll20/roll20-api-scripts/blob/master/SimpleSound/Readme.md) for full instructions",
```
Link to Readme + show some commands makes it easier to get started.